etc/aqd.conf.defaults

```diff
@@ -345,6 +345,8 @@ gzip_output = false
 # Assume the webserver will decompress transparently as needed.
 # only used if gzip_output = true
 transparent_gzip = true
+# Generate the template library variant of the plenary object template if true
+template_library_configured = false
```

Comment:
As per my suggestion change this option name as this is not obvious name for it.